mozregression/bisector.py

```diff
@@ -231,6 +231,26 @@ def user_exit(self, mid):
         LOG.info('%s known bad inbound revision: %s'
                  % (words[1], self.bad_revision))
 
+    def choose_integration_branch(self, changeset):
+        """
+        Tries to determine which integration branch the given changeset
+        originated from by checking the date the changeset first showed up
+        in each repo. The repo with the earliest date is chosen.
+        """
+        landings = dict()
+        # NB: Leaving b2g-inbound out for simplicity, it 404s at this point.
+        for k in ("autoland", "fx-team", "mozilla-inbound"):
+            jp = JsonPushes(k);
+
+            try:
+                push = jp.push(changeset, full='1')
+                landings[k] = push.timestamp
+            except EmptyPushlogError:
+                LOG.debug("Didn't find %s in %s" % (changeset, e))
```

Comment:
This is generating a flake8 error because you haven't defined e.

Try: `except EmptyPushLogError as e:` (`except EmptyPushLogError, e:` will also work, but the former is better because it's python 3 compatible)